CMakeLists.txt

```diff
@@ -455,6 +456,7 @@ if(BUILD_NVFUSER_BENCHMARK)
     # only install nvfuser_bench with submodule build
   else()
     torch_compile_options(${NVFUSER_BENCHMARK})
+    target_include_directories(${NVFUSER_BENCHMARK} PRIVATE ${TORCH_ROOT}/third_party/flatbuffers/include)
```

Comment:
looking forward to a day when we can just remove the else-block in `PROJECT_IS_TOP_LEVEL`